nodes/transaction-resubmission.md

```diff
@@ -0,0 +1,32 @@
+---
+description: This is a guide on transaction resubmission on Celestia.
+prev:
+  text: "Helpful CLI commands"
+  link: "/nodes/celestia-app-commands"
+---
+
+# Transaction resubmission
+
+In cases where transactions are not included within a 75-second window,
+resubmission is necessary. This is especially important during network
+congestion, as transactions with relatively low fees may not be processed
+even after the network clears up.
+
+Transactions, whether originating from celestia-app or celestia-node,
+won't be regossiped unless there's a new peer, even by nodes with longer TTLs.
+
+## Monitoring and resubmission
+
+Monitor the status of your transactions. If a transaction is not included within
+the 75-second window, it should be resubmitted. This can be done manually or
```

Comment:
[question] @cmwaters where does the 75 second window come from? Is it a param on the mempool? The only param I could find with a value of 75 is https://github.com/celestiaorg/celestia-app/blob/58b28cfe8ea0f6c9b2e114d7c335e074891267ab/specs/src/specs/params.md?plain=1#L41 but that doesn't appear immediately relevant.

```suggestion
a 75-second window, it should be resubmitted. This can be done manually or
```

Reply:
also, maybe I can link to this config?